app/sliders/plotter.py

```diff
@@ -14,6 +14,7 @@
     "bar": "BarPlotter",
     "bar_interactive": "InteractiveBarPlotter",
     "bubble": "BubblePlotter",
```

Comment:
Do you still want to keep the old unused `"bubble"` stuff around as well?

Reply:
No, I will get rid of it as well as the other obsolete classes during the next refactoring :)